modules/claims_api/spec/fixtures/usability_testing_claims.json

```diff
@@ -0,0 +1,40 @@
+{
+    "data": [
+        {
+            "id": "600098193",
+            "type": "evss_claims",
+            "attributes": {
+                "evss_id": 600098193,
+                "date_filed": "2017-05-02",
+                "min_est_date": null,
+                "max_est_date": null,
+                "open": false,
+                "waiver_submitted": false,
+                "documents_needed": false,
+                "development_letter_sent": false,
+                "decision_letter_sent": false,
```

Comment:
i was about to sign off and then i realized, these will return snake_case wont they? dont we need the keys here to be defined with camelCase?

Reply:
Do we?
This is just a copy/paste of what came out of prod.  (trimmed down to only include 2 items)

Reply:
yeah, i think so at least. the content is something we'll have to dig deeper into for sure, but the structure should be consistently camelCase throughout when people are testing it i feel like

Reply:
[Switched here](https://github.com/department-of-veterans-affairs/vets-api/pull/6955/commits/60e1207e4f0b799c386f08749d13f685b28839bc).